dds/DCPS/Service_Participant.cpp

```diff
@@ -2021,6 +2028,23 @@ void Service_Participant::default_configuration_file(const ACE_TCHAR* path)
   default_configuration_file_ = path;
 }
 
+NetworkConfigPublisher_rch Service_Participant::network_config_publisher()
+{
+  if (!network_config_publisher_) {
+#ifdef ACE_LINUX
+    network_config_publisher_ = make_rch<LinuxNetworkConfigPublisher>(reactor_task_.interceptor());
+#endif
+  }
```

Comment:
I don't think it would matter to the compiler, but I think it would look better if you put the ifdef around the the if block.

Reply:
Eventually there will be blocks for other platforms.  The `if` still needs to wrap all of it.